org.eclipse.pde.build/META-INF/MANIFEST.MF

```diff
@@ -26,6 +26,8 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="[3.2.0,4.0.0)",
  org.eclipse.equinox.p2.repository.tools;bundle-version="[2.0.0,3.0.0)";resolution:=optional,
  org.eclipse.equinox.p2.director.app;bundle-version="1.0.200",
  org.eclipse.equinox.p2.publisher.eclipse;bundle-version="1.0.0"
+ org.eclipse.jdt.core;bundle-version="[3.28.0,4.0.0)",
+ org.eclipse.jdt.launching;bundle-version="[3.2.0,4.0.0)"
```

Comment:
fyi - this is the only change that was required for the back port on top of @HannesWell 's original PR

from `org.eclipse.jdt.launching;bundle-version="[3.19.700,4.0.0)"` 
to `org.eclipse.jdt.launching;bundle-version="[3.2.0,4.0.0)"`

to match what is in there for `4.23`

Reply:
for that reason I intent to keep the second commit as is rather than squashing, but let me know if you think otherwise. (I understand that each project may have different rules)

Reply:
That's reasonable, would keep it separated too.